src/app/core/entity-components/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -60,6 +60,11 @@ export class EntitySubrecordComponent implements OnInit, OnChanges {
    */
   @Input() detailsComponent: ComponentType<ShowsEntity>;
 
+  /**
+   * Whether the records can be edited directly in the table.
+   */
+  @Input() inlineEditable: boolean = true;
```

Comment:
I think `editable` or `allowEditing` would be more clear.

Reply:
"inline" because there is also the functionality to open a popup and edit an entry there, which is independent of the editing inline as a table row directly

Reply:
would `disableEditingInline` be more readable? that would also be more similar to the broadly used `disabled` property

Reply:
Yes I think thats a good idea

Reply:
Yes I think thats a good idea